tests/PE/sections/empty-pe32-dll.test

```diff
@@ -0,0 +1,90 @@
+# RUN: %yaml2obj %s -o %t.obj
+# RUN: %bloaty --raw-map %t.obj | %FileCheck %s --dump-input fail
+
+--- !COFF
+OptionalHeader:
```

Comment:
How is this optional header reflected in the map?

I was hoping to report these as `[PE Image Headers]` or similar.

Reply:
It is included in the `[PE Headers]` entry.
For a PE file there are always:
- DOS Header
- Optional DOS stub (between DOS and NT header)
- NT Headers (See IMAGE_OPTIONAL_HEADER32 and IMAGE_OPTIONAL_HEADER64 in winnt.h):
  - Signature
  - FileHeader
  - OptionalHeader (contradicting it's name, it is not optional)

Reply:
@haberman is that good enough, or do you want me to rename `[PE Headers]` to `[PE Image Headers]`?

Reply:
> OptionalHeader (contradicting it's name, it is not optional)

I don't think that's entirely true: I believe it's required on DLLs and executables, but not on object files. It's the equivalent of what ELF calls "program headers."

In my comment above I was thinking of splitting the optional headers out into something like `[Image Headers]`. But since PE is having a lot more header types overall, it might not be worth splitting this out. You've already split out the section headers, which is the most important part.

So I think this is great as is.